src/shared/varlink-io.systemd.Machine.c

```diff
@@ -28,6 +28,11 @@ static SD_VARLINK_DEFINE_STRUCT_TYPE(
                 SD_VARLINK_FIELD_COMMENT("Timestamp in µs in the CLOCK_MONOTONIC clock"),
                 SD_VARLINK_DEFINE_FIELD(monotonic, SD_VARLINK_INT, SD_VARLINK_NULLABLE));
 
+static SD_VARLINK_DEFINE_METHOD(
+                Unregister,
+                SD_VARLINK_FIELD_COMMENT("The name of a machine to unregister."),
+                SD_VARLINK_DEFINE_INPUT(name, SD_VARLINK_STRING, 0));
```

Comment:
needs `SD_VARLINK_DEFINE_INPUT(allowInteractiveAuthentication, SD_VARLINK_BOOL, SD_VARLINK_NULLABLE)` line

Reply:
also, strictly speaking you allo unregistering via pid here too, not just name, hence pid field should be declared too, and name should be marked nullable